src/components/Stats/Volume/Stats.volume24.stories.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withKnobs } from '@storybook/addon-knobs'
+import Volume from './Volume'
+
+storiesOf('position/Stats', module)
+  .addDecorator(withKnobs)
+  .add('volume24', () => {
+    const [percerentVolume] = React.useState<number>(1.14)
+    const [volume] = React.useState<number>(231258435934)
+    const [data] = React.useState([
+      {
```

Comment:
I don't think data would have such format, instead it might be just timestamp and one value; in this case only some timestamps will be used on bottom axis as values

Reply:
done